### PR TITLE
Add docs for the image pull secrets CO option

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Documentation

### Description

This PR adds docs for #1578. It documents the new CO option.

### CODE REVIEW

**Comment by scholzj** [comment]
@ppatierno Sorry, I didn't expected you to review it today. If you have anything to it feel free to let me know per email.

**Comment by ppatierno** [comment]
@scholzj no worries I did a review anyway but no comment for that ;)

File: documentation/book/con-customizing-pods.adoc [hunk 1]
```diff
@@ -23,6 +23,8 @@ NOTE: You might need to increase the grace period for very large Kafka clusters,
 |Defines a list of references to {ProductPlatformName} Secrets that can be used for pulling container images from private repositories.
 For more information about how to create a Secret with the credentials, see {K8sPullingImagesFromPrivateRegistries}.
 
+NOTE: When both the `STRIMZI_IMAGE_PULL_SECRETS` environment variable in Cluster Operator and `imagePullSecrets` option are specified, only the `imagePullSecrets` will be used and the `STRIMZI_IMAGE_PULL_SECRETS` will be ignored.
```
Comment (PaulRMellor) [inline documentation/book/con-customizing-pods.adoc]:
```suggestion
NOTE: When the `STRIMZI_IMAGE_PULL_SECRETS` environment variable in Cluster Operator and the `imagePullSecrets` option are specified, only the `imagePullSecrets` variable is used. The `STRIMZI_IMAGE_PULL_SECRETS` variable is ignored.
```

File: documentation/book/ref-operators-cluster-operator-configuration.adoc [hunk 2]
```diff
@@ -86,3 +86,9 @@ The `ImagePullPolicy` which will be applied to containers in all pods managed by
 The valid values are `Always`, `IfNotPresent`, and `Never`.
 If not specified, the {ProductPlatformName} defaults will be used.
 Changing the policy will result in a rolling update of all your Kafka, Kafka Connect, and Kafka Mirror Maker clusters.
+
+`STRIMZI_IMAGE_PULL_SECRETS`:: Optional.
+A comma-separated list of `Secret` names.
+The secrets referenced here should contain the credentials to the container registries where the container images should be pulled from.
```
Comment (PaulRMellor) [inline documentation/book/ref-operators-cluster-operator-configuration.adoc]:
```suggestion
The secrets referenced here contain the credentials to the container registries where the container images are pulled from.
```

File: documentation/book/ref-operators-cluster-operator-configuration.adoc [hunk 3]
```diff
@@ -86,3 +86,9 @@ The `ImagePullPolicy` which will be applied to containers in all pods managed by
...
+The secrets will be used in the `imagePullSecrets` field for all `Pods` created by the Cluster Operator.
```
Comment (PaulRMellor) [inline documentation/book/ref-operators-cluster-operator-configuration.adoc]:
```suggestion
The secrets are used in the `imagePullSecrets` field for all `Pods` created by the Cluster Operator.
```

File: documentation/book/ref-operators-cluster-operator-configuration.adoc [hunk 4]
```diff
@@ -86,3 +86,9 @@ The `ImagePullPolicy` which will be applied to containers in all pods managed by
...
+Changing this list will result in a rolling update of all your Kafka, Kafka Connect, and Kafka Mirror Maker clusters.
```
Comment (PaulRMellor) [inline documentation/book/ref-operators-cluster-operator-configuration.adoc]:
```suggestion
Changing this list results in a rolling update of all your Kafka, Kafka Connect, and Kafka Mirror Maker clusters.
```

**Pull request merged**